### PR TITLE
chore(docs): Vietnamese README translation

### PR DESCRIPTION
**Pull request by hoangsvit**
I have added Vietnamese support and configured other READMEs for that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
   </a>
 </p>
 
-English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.zh-hk.md) | [العربية](./i18n/README.ar-dz.md) | [Italiano](./i18n/README.it-it.md) | [Українська](./i18n/README.ua-ua.md) | [Español](./i18n/README.es-es.md) | [Português](./i18n/README.pt.md) | [日本語](./i18n/README.ja-ja.md) | [Türkçe](./i18n/README.tr-tr.md) | [हिंदी](./i18n/README.hn-in.md)
```
Comment (bluestreak01) [inline README.md:18]: thank you for your contribution!

`i18n` paths must be kept though

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bluestreak01) [reply README.md:18]: you can test manually by clicking around links on https://github.com/hoangsvit/questdb/tree/master

Comment (hoangsvit) [reply README.md:18]: I fixed

**Comment by bluestreak01** [comment]
thanks 👍

**Pull request merged**